ipamd/ipamd.go

```diff
@@ -315,26 +322,25 @@ func (c *IPAMContext) nodeInit() error {
 		retry := 0
 		for {
 			retry++
-			err = c.setupENI(eni.ENIID, eni)
+			if err = c.setupENI(eni.ENIID, eni); err == nil {
+				log.Infof("ENI %s set up.", eni.ENIID)
+				break
+			}
+
 			if retry > maxRetryCheckENI {
```

Comment:
not related to this PR, seems if this branch `if retry > maxRetryCheckENI` triggered it will hide the real err?

Reply:
agreed. can fix that in followup.